src/apps/explorer/components/SummaryCardsWidget/VolumeChart/useGetVolumeData.ts

```diff
@@ -38,7 +38,7 @@ export function useGetVolumeData(volumeTimePeriod = VolumePeriod.DAILY): VolumeD
 }
 
 async function getLastHoursData(network: Network): Promise<RawVolumeItem[]> {
-  const data = await COW_SDK.cowSubgraphApi.getLastHoursVolume(48, { chainId: network })
+  const data = await subgraphApiSDK.getLastHoursVolume(48, { chainId: network })
 
```

Comment:
One thing I just realized as I changed this yesterday: Subgraph also has staging endpoints.
True, it's configured per environment with env vars, and maybe we don't need to change anything here.
Just for you to be aware

Reply:
Thanks! Checked staging and prod - they use the same endpoints